easybuild/easyconfigs/p/Panoply/Panoply-5.5.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PackedBinary'
+
+name = 'Panoply'
+version = '5.5.1'
+
+homepage = 'https://www.giss.nasa.gov/tools/panoply'
+description = "Panoply plots geo-referenced and other arrays from netCDF, HDF, GRIB, and other datasets."
+
+
+toolchain = SYSTEM
+
+source_urls = ['https://www.giss.nasa.gov/tools/panoply/download/']
+sources = ['%(name)sJ-%(version)s.tgz']
+checksums = ['14196be2dd83721e475dfa0b230859b9e102bfdcc62119536ececbf8dcbbbb96']
+
+dependencies = [
+    ('Java', '11', '', SYSTEM),
+]
+
+postinstallcmds = [
+    'mkdir %(installdir)s/bin',
+    'mv %(installdir)s/panoply.sh %(installdir)s/bin',
+    'sed -i "s,jars,../jars,g" %(installdir)s/bin/panoply.sh',
+    'ln -s %(installdir)s/bin/panoply.sh %(installdir)s/bin/panoply',
+]
+
+sanity_check_paths = {
+    'files': ['bin/panoply'],
```

Comment:
@WilleBell Can we add a sanity check command for this? Like `panoply --help` or something?

Reply:
I couldn't find anything that would serve as sanity check, it all opens the GUI.

Reply:
OK, nevermind then